src/nrnpython/nrnpy_p2h.cpp

```diff
@@ -118,7 +118,6 @@ static void call_python_with_section(Object* pyact, Section* sec) {
         if (mes) {
             Fprintf(stderr, "%s\n", mes);
             free(mes);
-            lock.release();
```

Comment:
These calls are no longer needed because now the destructor of `lock` is called.